collect_app/src/main/java/org/odk/collect/android/activities/FormEntryActivity.java

```diff
@@ -2478,6 +2478,12 @@ public void denied() {
                 activityDisplayed();
 
                 refreshCurrentView();
+
+                if (warningMsg != null) {
+                    String msg = getString(warningMsg);
+                    ToastUtils.showLongToast(msg);
+                    Timber.e(msg);
```

Comment:
Since this is an issue that has to do with the structure of the given CSV file rather than with the code, I don't think it should be logged as an error.

Reply:
Right, fixed.